onnxruntime/core/providers/tensorrt/symbols.txt

```diff
@@ -1 +1,5 @@
 OrtSessionOptionsAppendExecutionProvider_Tensorrt
+OrtCreateTensorRTProviderOptions
```

Comment:
Yeah, don't do this. We should only export new functions through the C API. We want to remove the existing function in the future as we can't version it, and it's not always present.

Reply:
Moved these functions in the C API as well.